app/assets/javascripts/mpa/processors/peptide/TaxaPeptideProcessor.ts

```diff
@@ -0,0 +1,29 @@
+import {ProcessedPeptideContainer} from '../../ProcessedPeptideContainer';
+import {Pept2DataResponse} from '../../api/pept2data/Response';
+import {TaxaCountTable} from '../../counts/TaxaCountTable';
+import {Count} from '../../counts/CountTable';
+
+export namespace TaxaPeptideProcessor
```

Comment:
Minor remark: Why don't we make a class of this instead of a namespace with a single function? The same remark holds for `GOPeptideProcessor` and `ECPeptideProcessor`

Reply:
Because the processors have no state.